src/r2mm/launching/instructions/DynamicGameInstruction.ts

```diff
@@ -2,6 +2,7 @@ export enum DynamicGameInstruction {
     BEPINEX_PRELOADER_PATH = "@bepInExPreloaderPath",
     PROFILE_DIRECTORY = "@profileDirectory",
     BEPINEX_CORLIBS = "@bepInExCorlibs",
+    SKUL_THE_HERO_SLAYER_CORELIBS = "@skulTheHeroSlayerCorlibs",
```

Comment:
Any way to make this not be skul specific? Can the `BEPINEX_CORLIBS` perhaps be re-used instead (AFAIK there shouldn't be anything special about how skul handles them right?)